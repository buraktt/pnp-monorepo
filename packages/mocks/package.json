{
  "name": "@monorepo/mocks",
  "version": "0.0.1"
}
