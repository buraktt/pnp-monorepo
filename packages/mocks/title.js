export const titleMocks = {
  title: 'Title test',
  color: 'primary'
}
