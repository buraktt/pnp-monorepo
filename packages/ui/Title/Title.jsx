import React from 'react'
import { Typography } from '@material-ui/core'

export default function Title(props) {
  const { title, color } = props
  return <Typography color={color}>{title}</Typography>
}
