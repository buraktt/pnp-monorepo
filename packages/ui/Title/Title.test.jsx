/**
 * @group snapshot
 */
import React from 'react'
import Title from './Title'
import { render } from '@testing-library/react'
import { titleMocks } from '@monorepo/mocks/title'

describe('<Title />', () => {
  it('should render without errors', () => {
    const { asFragment } = render(<Title {...titleMocks} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
