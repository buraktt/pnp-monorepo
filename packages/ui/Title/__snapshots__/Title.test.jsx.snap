// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render without errors 1`] = `
<DocumentFragment>
  <h1
    class="MuiTypography-root MuiTypography-h1 MuiTypography-colorPrimary"
  >
    Title test
  </h1>
</DocumentFragment>
`;
